packages/compass-aggregations/src/utils/stage.js

```diff
@@ -103,4 +105,55 @@ export function getStageOperator(stage) {
     return typeof ns === 'object' ? `${ns.db}.${ns.coll}` : `${database}.${ns}`;
   }
   return null;
+}
+
+const OutOperatorNames = new Set(OUT_STAGES.map(stage => stage.value));
```

Comment:
NIT: shouldn't this be "constant case"? I think we used this camel case for class names and enums only

Reply:
I always think about these as static / singleton classes (it is an instance with methods, but it never changes after initial creation), so that's why I went with this casing. But it might be just me, will update